tools/querytee/proxy_endpoint.go

```diff
@@ -165,7 +165,7 @@ func (p *ProxyEndpoint) executeBackendRequests(r *http.Request, resCh chan *back
 		err := p.compareResponses(expectedResponse, actualResponse)
 		if err != nil {
 			level.Error(util_log.Logger).Log("msg", "response comparison failed", "route-name", p.routeName,
-				"query", r.URL.RawQuery, "err", err)
+				"query", r.URL.RawQuery, "org-id", r.Header.Get("X-Scope-OrgID"), "err", err)
```

Comment:
[nit] Just to keep consistency with Mimir logs.

```suggestion
				"query", r.URL.RawQuery, "user", r.Header.Get("X-Scope-OrgID"), "err", err)
```

Reply:
I'm self-committing this. I hope you don't mind.

Reply:
Not at all, thanks for making this change.